setup.py

```diff
@@ -18,7 +18,8 @@
     'docker',
     'pyyaml',
     'pygithub',
-    'catkin_pkg >= 0.4.0'
+    'catkin_pkg >= 0.4.0',
+    'bs4'
```

Comment:
This module addition is causing CI failure. Please add this package to the installed list in the `.travis-ci.yml` file.

Reply:
@allenh1  << Thanks a bunch for your thorough review!! I'll rebase and push the small changes/refactors separately as suggested.

I'll update requirements.txt which is referenced by travis.yml .

Reply:
@andre-rosa that would be great. Thanks!

Reply:
Done; thanks!